frontend/src/components/PlannerChart.tsx

```diff
@@ -168,53 +197,60 @@ export const PlannerChart: FC<{
               type="linear"
               dataKey={DataKeys.OptimalRoadmap}
               stroke="black"
+              strokeWidth={2}
             />
           )}
-          {versionKeyNames.map((vername, index) => (
+          {versionKeyNames.reverse().map((vername, index) => (
             <Line
               isAnimationActive={false}
               id={vername}
               key={vername}
               name={vername}
               type="linear"
               dataKey={vername}
-              stroke={colors[index % colors.length]}
+              stroke={
+                colors[(versionKeyNames.length - 1 - index) % colors.length]
+              }
+              strokeWidth={2}
```

Comment:
I'd change the strokeWidth to something like 1.5, this value 2 looks a bit too thick in my opinion. And those other two places as well where to strokeWidth was at 2